### PR TITLE
fix eas card styling - buttons shifting TM-3355

### PR DESCRIPTION
**Pull request by SidelineCory24**
To-do

- [x] buttons in card should not shift when data points spill over to 2-3 lines
- [x] verify this works for AO
- [x] verify this works for CDO

Name to test with: `WEETHERS-DUMANCASEE,WUNLOO-ADDITEE`

### CODE REVIEW

**Comment by SidelineCory24** [comment]
before
image

after
image

**Comment by elizabeth-jimenez** [comment]
This looks so nice 💅 !

File: src/sass/_employeeSearch.scss [hunk 1]
```diff
@@ -221,7 +221,8 @@
     }
 
     .employee-agenda-card-inner {
```
Comment (elizabeth-jimenez) [inline src/sass/_employeeSearch.scss:223]: Only suggestion is that we can eliminate two classes(`employee-agenda-card-top` and `employee-ribbon-container`) by doing this instead:

    ```
.employee-agenda-card-inner {
      padding-right: 16px;
      padding-top: 16px;
      position: relative;
      display: flex;
      .ribbon-container-condensed {
        position: absolute;
        right: 0;
        top: 0;
        z-index: 2;
      }
    }
```

Likewise, we'll be able to remove the div that has the class `employee-agenda-card-top` and the one that has the `employee-ribbon-container` class in `EmployeeAgendaSearchCard.jsx`

Comment (SidelineCory24) [reply src/sass/_employeeSearch.scss:223]: done

**Review by elizabeth-jimenez — APPROVED** [review]
🚀

**Pull request merged**